x-pack/plugins/security_solution/public/timelines/store/timeline/reducer.ts

```diff
@@ -177,12 +178,12 @@ export const timelineReducer = reducerWithInitialState(initialTimelineState)
     ...state,
     timelineById: addTimelineNoteToEvent({ id, noteId, eventId, timelineById: state.timelineById }),
   }))
-  .case(toggleExpandedEvent, (state, { timelineId, event }) => ({
+  .case(toggleExpandedEvent, (state, { timelineId, event, defaultModel }) => ({
     ...state,
     timelineById: {
       ...state.timelineById,
       [timelineId]: {
-        ...state.timelineById[timelineId],
+        ...(state.timelineById[timelineId] ?? defaultModel ?? timelineDefaults),
```

Comment:
it shoud

Reply:
I'm removing all the defaultModel from action and in reducer, but somehow in cypress test `alerts.spec.ts`, the default `showCheckboxes` on detections-page is incorrectly set to `false`, and therefor the tests failed.